deploy/foreman-tasks.cron.d

```diff
@@ -0,0 +1,8 @@
+SHELL=/bin/sh
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+
+RAILS_ENV=production
+FOREMAN_HOME=/usr/share/foreman
+
+# Clean up the session entries in the database
+45 19 * * *     foreman    /usr/sbin/foreman-rake foreman_tasks:cleanup >>/var/log/foreman/cron.log 2>&1
```

Comment:
As was said in PR description: The time was set to 7:45 pm so it happens before nightly syncs
and do not collide with other default tasks of Foreman. Feel free to suggest more reasonable time.